security/org-policies/dependencies.tf

```diff
@@ -333,7 +333,51 @@ POLICY
           ]
         }
       }
-    }
+    },
+    {
+            "Sid": "DenyCostManagement",
+            "Effect": "Deny",
+            "Action": [
+                "ce:*",
+                "savingsplans:*"
+            ],
+            "Resource": [
+                "*"
+            ],
+            "Condition": {
+                "StringNotLike": {
+                    "aws:PrincipalArn": [
+                        "arn:aws:iam::*:role/OrgRole",
+                        "arn:aws:iam::*:role/aws-reserved/sso.amazonaws.com/*/AWSReservedSSO_CloudAdmin_*",
+                        "arn:aws:iam::*:role/aws-reserved/sso.amazonaws.com/*/AWSReservedSSO_Billing_*"
+                    ]
+                }
+            }
+        },
+        {
+            "Sid": "DenyBillingLegacyRemoveAfterJuly2023",
```

Comment:
Can you put a note here on what the plan is with this policy? Why is it being removed?

Reply:
Jan want to deny all teams to view their billing data. I think the idea is to force them to use a FinOps tool we will subscribe to for 1 year.

Reply:
Also, the IAM policies for billings are being replaced with new ones. Ref: https://docs.aws.amazon.com/cost-management/latest/userguide/migrate-granularaccess-whatis.html#migrate-user-permissions

Reply:
I see, I think it would be good if these types of plans were documented near the relevant code as comments.